plugins/openchoreo-observability/src/api/ObservabilityApi.ts

```
@@ -266,44 +282,47 @@ export class ObservabilityClient implements ObservabilityApi {
-      },
+      )}`,
+    );
+    url.searchParams.set('environmentUid', environmentId);
+    if (options?.startTime)
+      url.searchParams.set('startTime', options.startTime);
+    if (options?.endTime) url.searchParams.set('endTime', options.endTime);
+    if (options?.status) url.searchParams.set('status', options.status);
+    if (options?.limit !== undefined)
+      url.searchParams.set('limit', String(options.limit));
+    if (componentUids.length > 0) {
+      componentUids.forEach(uid =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)